types/parcel/index.d.cts

```diff
@@ -29,5 +29,8 @@ declare class Parcel<ExtraProps = {}> extends React.Component<
   ParcelState
 > {}
 
-// @ts-ignore
 export = Parcel;
+
+declare namespace Parcel {
```

Comment:
This could be breaking, please don't add additional stuff to this after I have reviewed a PR